CheckDrive.Web/CheckDrive.Web/Views/Dashboard/Index.cshtml

```diff
@@ -103,27 +103,21 @@
         </div>
     </div>
 </div>
-
 <div class="row">
     <div class="col">
-        <div class="widget">
-            <div class="p-4">
-                <h5 class="fw-bold">Qarzlar</h5>
-            </div>
-            <div class="px-4 pb-4">
+            <div style="margin-bottom:10px;">
                 <ejs-grid id="resent-debts" allowPaging="true"
-                          dataSource="@ViewBag.Debts"  gridLines="None" rowHeight="38">
+                          dataSource="@ViewBag.CheckPoint" gridLines="None">
```

Comment:
There is more space left on the left side of the checkpoint grid, is it possible to reduce this space?